AdminUi/src/AdminUi.Infrastructure/DTOs/ClientOverview.cs

```diff
@@ -3,7 +3,8 @@ public class ClientOverview
 {
     public string ClientId { get; set; }
     public string DisplayName { get; set; }
-    public string DefaultTier { get; set; }
+    public string DefaultTierId { get; set; }
+    public string DefaultTierName { get; set; }
```

Comment:
```suggestion
    public TierDTO DefaultTier { get; set; }
```

Should be like that in all places, of course. I won't comment all of them.

Reply:
Are these changes only for the ClientOverview view?
The IdentityOverview view also has this format:
![image](https://github.com/nmshd/backbone/assets/118175875/99455a67-d9f4-45ab-8143-14544493e92a)

Should I change it there as well?